src/foam/u2/crunch/CrunchController.js

```diff
@@ -19,6 +19,7 @@ foam.CLASS({
     'ctrl',
     'prerequisiteCapabilityJunctionDAO',
     'stack',
+    'user',
```

Comment:
since here- can you make this import 'subject'

this.user == this.subject.user